### ISSUE TITLE
Clear should hide the suggestion list, if present

### ISSUE THREAD

**Issue by tomasz-biernacki** [Lambda-X/cljs-repl-web#218]
See gif below:
![o0fixf0vel](https://cloud.githubusercontent.com/assets/15321488/14882592/d06bd3d0-0d39-11e6-8d23-44a391e3fc92.gif)

To check also Reset and Clear Examples

**Comment by arichiardi** [comment]
Yeah this is a classic examples that, if we did things right, we just need to dispatch an event
